libmscore/style.cpp

```diff
@@ -498,7 +498,7 @@ static const StyleType styleTypes[] {
       { Sid::ottavaLineStyle,         "ottavaLineStyle",         QVariant(int(Qt::DashLine)) },
       { Sid::ottavaNumbersOnly,       "ottavaNumbersOnly",       true },
       { Sid::ottavaFontFace,          "ottavaFontFace",          "FreeSerif" },
-      { Sid::ottavaFontSize,          "ottavaFontSize",          12.0 },
+      { Sid::ottavaFontSize,          "ottavaFontSize",          10.0 },
```

Comment:
As only non-default style settings get written to .mscx changing this setting would probably affect appearance of existing scores. Is this what is intended to be achieved here?

Reply:
yes it is